docs/mast/mast.rst

```diff
@@ -1353,6 +1413,25 @@ The basic MAST query function returns query results as an `~astropy.table.Table`
    Length = 77 rows
 
 
+Many mast services, specifically JWST and Catalog services, require the two principal keywords, columns and filters,
```

Comment:
I would put quotes around the keyword names, so formatted like this: _require the two principal keywords, "columns" and "filters", to list parameters_